modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-top-aware_{context}"]
+= vSphere Topology Awareness
+
+{product-title} provides the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple data centers and clusters, thus helping to avoid a single-point-of-failure.
+
+.Procedure
+Use the following procedure to deploy {product-title} for vSphere storage on different zones and regions.
+
+. In the VMware vCenter vSphere client GUI:
+.. Create and assign `openshift-zone` and `openshift-region` category tags as needed to the hosts on the desired clusters.
```

Comment:
Switch `category tags` to `tag categories`
link vmware's docs to create tags and tag categories 

https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vcenterhost.doc/GUID-E8E854DD-AA97-4E0C-8419-CE84F93C4058.html

Reply:
This section is confusing to me.
You don't assign tag categories to vcenter objects. You create tags that are associated with a category. The tag is attached to either a vcenter datacenter or cluster.

See the zonal presentation for 4.12 TP:
https://docs.google.com/presentation/d/1AAuGLYRQ93rCS-6nKrcHaBTNqt9wNvwkfNWR66n3i90/edit#slide=id.g14cc0ab5502_0_44

Reply:
Fixed.

Reply:
There is a link to vwmare's doc for tags/categories in the Additional Resources section.